Bluepill-runner/Bluepill-runner/BPReportCollector.h

```diff
@@ -9,6 +9,25 @@
 
 #import <Foundation/Foundation.h>
 
+@interface TraceEvent  : NSObject
```

Comment:
Move to a new file and prefix with BP (objc convention to avoid naming conflicts)